assembly/__tests__/env-exec.spec.ts

```diff
@@ -0,0 +1,55 @@
+// This file is aim to test the env execution related functions which are external functions.
+// For now we only simple test the functions that are not using the call stack.
+
+import { env } from '../env';
+import { Address, Context } from '../std';
+
+const testAddress = new Address(
+  'AU12E6N5BFAdC2wyiBV6VJjqkWhpz1kLVp2XpbRdSnL1mKjCWT6oR',
+);
+
+describe('Testing env coins related functions', () => {
+  test('Testing localExecution (assembly_script_local_execution)', () => {
```

Comment:
Same as in the other review (test + Testing and the name of the internal function that is useles I guess).

Reply:
you're right, I change that